surfkit.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/helloink/surfkit/events"
 )
 
-const version = "1.9.0"
+const version = "1.9.1"
```

Comment:
This should be `1.10.0`. Patch numbers are supposed to contain bug fixes for a minor version, see [semver](https://semver.org). Same for the change in `changelog`

Reply:
Good point. I've changed it to 1.10.0.